sequence_processing_pipeline/templates/telllink-isolate.sbatch

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -l
+#SBATCH -J {{job_name}}             # tellink-isolate
+#SBATCH -N {{node_count}}           # 1
+#SBATCH -c {{cores_per_task}}       # 16
+#SBATCH --mem {{mem_in_gb}}G        # 160G
+#SBATCH --time {{wall_time_limit}}  # 96:00:00
+#SBATCH -p {{queue_name}}           # qiita
+
+#SBATCH --output {{output}}/logs/telllink-isolate_%x-%A_%a.out
+#SBATCH --error {{output}}/logs/telllink-isolate_%x-%A_%a.err
+
+set -x 
+set -e
+
+module load {{modules_to_load}}
+
+samples=($(cat {{output_path}}/sample_index_list_output.txt | cut -f 2))
+sample=${samples[$((${SLURM_ARRAY_TASK_ID} - 1))]} 
+
+k=79
+lc=35
```

Comment:
Should these be parameters in the configurations files?

Reply:
Agreed but this is part of assemblies. This is a holdover from the original porting.